src/store/modules/loadingStatus.ts

```diff
@@ -0,0 +1,30 @@
+import { Action, Module, Mutation, VuexModule } from 'vuex-module-decorators'
+
+@Module({ namespaced: true })
+export default class loadingStatusModule extends VuexModule {
+  // to show progress used in app.vue
+  globalLoader: boolean = false
+  // to show progress in an individual component
+  localLoader: boolean = false
```

Comment:
is it using somewhere? good to have isLoading than localLoader

Reply:
no. we dont use it. local loading is achevied using isLoading is useLoader. So removed it